### PR TITLE
minor LUA and trait documentation fixes, whitespace fixes

### PR DESCRIPTION
**Pull request by clemty**
- Typo in documentation ("proximitry").
- Add spaces between sentences.
- Remove most occurrences of "  " (two spaces) unless clearly used as indendation
- Punctuation (although no fullstop after filenames like "notifications.yaml").

### CODE REVIEW

File: OpenRA.Game/FileFormats/Blast.cs [hunk 1]
```diff
@@ -219,10 +219,10 @@ public int ReadBits(int count)
 	/*
 	 * Given a list of repeated code lengths rep[0..n-1], where each byte is a
 	 * count (high four bits + 1) and a code length (low four bits), generate the
-	 * list of code lengths.  This compaction reduces the size of the object code.
+	 * list of code lengths. This compaction reduces the size of the object code.
 	 * Then given the list of code lengths length[0..n-1] representing a canonical
 	 * Huffman code for n symbols, construct the tables required to decode those
-	 * codes.  Those tables are the number of codes of each length, and the symbols
+	 * codes. Those tables are the number of codes of each length, and the symbols
```
Comment (Mailaender) [inline OpenRA.Game/FileFormats/Blast.cs]: I believe @pchote adding these double spaces on purpose.

**Comment by abcdefg30** [comment]
:+1: with above answered.

**Comment by reaperrr** [comment]
Needs rebase.

**Comment by clemty** [comment]
reverted double spaces in comments & rebased

File: OpenRA.Game/Map/CellLayer.cs [hunk 2]
```diff
@@ -155,7 +155,7 @@ public MPos Clamp(MPos uv)
 	// Helper functions
 	public static class CellLayer
 	{
-		/// <summary>Create a new layer by resizing another layer.  New cells are filled with defaultValue.</summary>
```
Comment (reaperrr) [inline OpenRA.Game/Map/CellLayer.cs]: This might have been intentional as well.

**Comment by reaperrr** [comment]
:+1: once my comment has been adressed.

File: OpenRA.Game/Map/CellLayer.cs (hunk 2 above)
Comment (clemty) [inline OpenRA.Game/Map/CellLayer.cs]: this looks like html to me, and multiple whitespace characters get filtered on display? or are tags like these used by IDEs too?

Comment (penev92) [inline OpenRA.Game/Map/CellLayer.cs]: `<summary>` is an IDE thing for code documentation and easy lookup. But I'm voting for keeping your change anyway.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/2e1622a96544e2f691ca5d6d824e051f2d632097...0135622feaec14d6ba3a10d12aaecbcd6cad410f)

**Pull request merged**